src/popup/components/SeedPhraseNotification.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div :class="['seed-phrase-notification', { error: hasError }]">
+    <div class="icon-wrapper">
+      <Alert v-if="hasError" />
+      <CheckCircle v-else />
+    </div>
+    <div class="text">
+      <div>
+        {{
+          hasError
+            ? $t('pages.seed-phrase-settings.seed-phrase-incorrect')
+            : $t('pages.seed-phrase-settings.seed-phrase-correct')
+        }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api/dist/vue-composition-api';
+import CheckCircle from '../../icons/check-circle.svg?vue-component';
+import Alert from '../../icons/alert.svg?vue-component';
```

Comment:
I suggest to add `...Icon` to icon components as in Vue 3 there is no way of having one-word component names.

Same for the `CheckCircle` above.